src/main/webapp/ts/login.ts

```diff
@@ -2,20 +2,23 @@
 type authInfo = {
     loginUrl: string;
     logoutUrl: string;
+    hasProfile: boolean;
 };
 //function that loads the home page for logging in and logging out
 async function loadHome() {
     const logging = document.getElementById('navbarResponsive');
     const link = document.getElementById('login-link');
-
-    const logStatus = await getLogStatus();
-
+   
+    const logStatus = await logStatusMethod();
+    if(logStatus.hasProfile == false && logStatus.loginUrl == ""){
+        console.log(1);
```

Comment:
Can this console.log() method be removed (L14)?

Reply:
Removed that